obs.py

```diff
@@ -190,6 +192,11 @@ def __init__(self, name, config):
         self.aws_queue_thread = threading.Thread(target=self.send_to_aws, args=())
         self.aws_queue_thread.start()
 
+        self.fast_queue = queue.PriorityQueue(maxsize=0)
```

Comment:
The separate queues are a great solution for the jpg feedback, I'm not sure why this hadn't been done earlier!